docs/user.adoc

```diff
@@ -1521,7 +1521,7 @@ The list of available credentials will depend on a number of factors:
 
 [WARNING]
 ====
-Credentials parameters can only access the per-user credentials store of the job that was explicitly triggered by the user.
+Credentials parameters can only access user credentials if the corresponding user is the one who chooses them during a build.
```

Comment:
I'm not sure what this is saying. The phrasing is a little convoluted. It could be unwound for clarity.